.github/workflows/update.yaml

```diff
@@ -0,0 +1,41 @@
+name: Population-city pipeline
+
+on:
+  push:
+    branches:
+      - master
+      - population_city
```

Comment:
@judeleonard i don't think we need the update job to run on any branches other than master

Reply:
Updated